### PR TITLE
Page title component

### PR DESCRIPTION
**Pull request by ivoBW**
## Summary

* Separo el titile en un componente, le agrego un background decorativo
* Agrego el segundo titulo de la pagina

## Screenshots

image

## Trello Card

https://trello.com/c/nlY2t10J/21-page-titles

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -9,6 +9,8 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1>
+  <h1 class="title">Front-End Tools</h1>
+  <br>
```
Comment (jjuli93) [inline maquetado/index.html]: try not to use line breaks

File: maquetado/index.html [hunk 2]
```diff
@@ -9,6 +9,8 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1>
+  <h1 class="title">Front-End Tools</h1>
```
Comment (jjuli93) [inline maquetado/index.html]: it is good practice to have a single h1 for each page. Maybe you can use h2

File: maquetado/scss/styles.scss [hunk 3]
```diff
@@ -1,4 +1,10 @@
...
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+$main-font: 'roboto';
```
Comment (jjuli93) [inline maquetado/scss/styles.scss]: apply the font to the element.
`.* { font-family: $main-font }`

File: maquetado/scss/titles.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+@import 'styles';
+
+.title {
+  background-image: url(assets/triangle.svg);
```
Comment (jjuli93) [inline maquetado/scss/titles.scss]: you must use $title-decoration-height and $title-decoration-width for the image dimensions.
Try using the :before pseudo element

**Comment by ivoBW** [comment]
image

**Pull request merged**